include/dxc/Support/HLSLOptions.td

```diff
@@ -234,10 +234,18 @@ def no_min_precision: Flag<["-", "/"], "no-min-precision">, Flags<[CoreOption, D
 def ignore_line_directives : Flag<["-", "/"], "ignore-line-directives">, HelpText<"Ignore line directives">, Flags<[CoreOption]>, Group<hlslcomp_Group>;
 
 // SPIRV Change Starts
-def spirv : Flag<["-"], "spirv">, Group<spirv_Group>, Flags<[CoreOption, DriverOption, HelpHidden]>,
+def spirv : Flag<["-"], "spirv">, Group<spirv_Group>, Flags<[CoreOption, DriverOption]>,
   HelpText<"Generate SPIR-V binary code">;
 def fvk_stage_io_order_EQ : Joined<["-"], "fvk-stage-io-order=">, Group<spirv_Group>, Flags<[CoreOption, DriverOption, HelpHidden]>,
   HelpText<"Specify Vulkan stage I/O location assignment order">;
+def fvk_b_shift_EQ : Joined<["-"], "fvk-b-shift=">, Group<spirv_Group>, Flags<[CoreOption, DriverOption]>,
```

Comment:
I'd re-emphasize the SPIR-V-ness of these options with something along the lines of "...register number shift for SPIR-V b-type register" (same for other flags).

Reply:
Done! Changed to `Specify Vulkan binding number shift for t-type register` and similar.